### DISCUSSION TITLE
only in step 2 google AddressAutocomplete didnt work

### DISCUSSION THREAD

**Discussion by hai-shift** [spatie/laravel-livewire-wizard#42] [Q&A]
i have a problem
 when i use my component for google AddressAutocomplete
 somthing like that in the step component : 
<x:google-address-lookup wire:model="address"/>

inside the component:
(need to install https://www.npmjs.com/package/google-address-autocomplete)
```
<div class="input-group"
    wire:ignore
    x-data={lookup:{}}
    x-init="() => {
        new AddressAutocomplete('.google-address-lookup', (result, raw) => {
            @this.set('lookup', result)
        });
    }"
    {{ $attributes }}
    >
    <input
        x-data
        x-on:address:list:refresh.window="$('.google-address-lookup').val('')"
        x-on:keydown.enter.prevent
        wire:ignore.self
        wire:loading.attr="readonly"
        autocomplete="google-lookup"
        class="form-control google-address-lookup"
        required
        type="search"
    >
</div>
```

when i use in first step the component work and i see the address-autocomplete in dropdown and my chrome ask if i whant to use my locaction

but if i put the 
<x:google-address-lookup wire:model="address"/>
in the secound step its didnt work
and i didnt get an error in a console
when i start to type the address-autocomplete didnt work and the chrome didnt ask me to get my locaction

someone can help me with that problem?

thanks :)

**Comment by ju5t** [comment]
I do not think this is an issue with this package. Can you elaborate?

**Reply by hai-shift** [reply]
hi
i elaborate many i can 

i just want to run a google autocomplete JS with alpine
in step 2 on laravel-livewire-wizard
when i try in the step 1 its work
but when i use the component in step 2 is didnt work :/

**Reply by ju5t** [reply]
I can't debug third party implementations. Especially one that mixes Alpine and Bootstrap as that's generally a bad idea.

Have you tried a simplifying the component? E.g. try if `x-init` works by just using `console.log`? And go from there.
